src/test/regress/sql/pg16.sql

```diff
@@ -623,22 +604,13 @@ SELECT roleid::regrole::text AS role, member::regrole::text,
 admin_option, inherit_option, set_option FROM pg_auth_members
 WHERE roleid::regrole::text = 'role1' ORDER BY 1, 2;
 
-SET citus.enable_ddl_propagation TO off;
-GRANT role1 TO role2 WITH ADMIN OPTION, INHERIT FALSE, SET FALSE;
-RESET citus.enable_ddl_propagation;
-
-SELECT roleid::regrole::text AS role, member::regrole::text,
-admin_option, inherit_option, set_option FROM pg_auth_members
-WHERE roleid::regrole::text = 'role1' ORDER BY 1, 2;
-
 \c - - - :master_port
-REVOKE role1 FROM role2;
 
 -- test REVOKES as well
 GRANT role1 TO role2;
 REVOKE SET OPTION FOR role1 FROM role2;
 REVOKE INHERIT OPTION FOR role1 FROM role2;
```

Comment:
I think we should add the propagated commands in the output here. It shows that these commands work on the current node without verifying that they actually got propagated to worker nodes.

I think it is sufficient to set some GUCs to log remote commands and filter on REVOKE commands by using the following statements before running these REVOKE commands.

SET citus.log_remote_commands TO on;
SET citus.grep_remote_commands = '%REVOKE%';